### DISCUSSION TITLE
bounce effect

### DISCUSSION THREAD

**Discussion by qp5** [DevsOnFlutter/draggable_home#20] [General]
Hi, 
I'm considering your Responsive scroll, for App in development.  
Noticed from <a href="https://camo.githubusercontent.com/2f60416a3c59ad8568cd5abca18d86ab5cb1e488c13e2b3118fbb753af1e9953/68747470733a2f2f696d6775722e636f6d2f6c4174675530452e676966">preview image</a> swiping down on screen (to get back to top) creates bounce effect.

Question, 
Does code provider option for developer to increase / decrease amount of bounce effect?

**Comment by 4-alok** [comment]
The bouncing effect is created by BouncingScrollPhysics(), however you may adjust the scrolling trigger point by `stretchTriggerOffset`.

**Comment by qp5** [comment]
Thank you for your response!

Will plan to integrate code.

Cool idea.

Thank you
